app/views/foreman/smart_proxies/_content_sync.html.erb

```diff
@@ -21,15 +21,32 @@
   </div>
 
   <br>
-
-  <div>
-    <a ng-show="syncState.is(syncState.SYNCING, syncState.SYNC_TRIGGERED, syncState.CANCEL_TRIGGERED)"
-      ng-click="cancelSync()" class="btn btn-default" ng-disabled="!syncState.is(syncState.SYNCING)">
+  <div ng-show="syncState.is(syncState.SYNCING, syncState.SYNC_TRIGGERED, syncState.CANCEL_TRIGGERED)">
+    <a ng-click="cancelSync()" class="btn btn-default" ng-disabled="!syncState.is(syncState.SYNCING)">
       <span translate>Cancel Sync</span>
     </a>
-    <a ng-hide="syncState.is(syncState.SYNCING, syncState.SYNC_TRIGGERED, syncState.CANCEL_TRIGGERED)"
-      ng-click="syncCapsule()" class="btn btn-default">
-      <span translate>Synchronize</span>
-    </a>
+  </div>
+  <div class="dropdown" ng-hide="syncState.is(syncState.SYNCING, syncState.SYNC_TRIGGERED, syncState.CANCEL_TRIGGERED)">
+    <button class="btn btn-default dropdown-toggle" type="button" id="syncDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+      Synchronize
+      <span class="caret"></span>
+    </button>
+    <ul class="dropdown-menu" aria-labelledby="syncDropdown">
```

Comment:
@bbuckingham that one is a bit more difficult to implement so I just added this option for now, I think it would be best to follow up with the validate option in a later PR if we deem it necessary